Fitness/Controllers/HabitTrackingController.swift

```diff
@@ -247,7 +254,7 @@ class HabitTrackingController: UIViewController {
         habitTableView.snp.makeConstraints { make in
             make.leading.trailing.equalToSuperview()
             make.top.equalTo(descriptionLabel.snp.bottom).offset(8)
-            let height = 68 + HabitTrackerOnboardingCell.height * ( habits.count + 1 )
+            let height = 84 + HabitTrackerOnboardingCell.height * ( habits.count + 1 )
```

Comment:
make `84` a constant at the beginning of this function. also make a comment explaining why we do `habits.count + 1`

Reply:
yup!